build_helpers/cssRollup.config.js

```diff
@@ -0,0 +1,84 @@
+import multiEntry from '@rollup/plugin-multi-entry';
+import postcss from 'rollup-plugin-postcss';
+import path from 'path';
+import autoprefixer from 'autoprefixer';
+import postcssCustomProperties from 'postcss-custom-properties';
+import cssVar from '../src/stubs/cssVar';
+
+function escapeSlash(match) {
+  return match.replace(/\//g, '_');
+}
+
+function slashTransform(content) {
+  return content.replace(/\.[\w\/\:\.]+(\s|\,)/g, escapeSlash);
+}
+
+// NOTE (pradeep): Our custom rollup plugin that transforms slashes to underscores in CSS files
+// Check https://rollupjs.org/guide/en/#plugins-overview for a quick start on roll plugins
+function cssSlashTransformPlugin() {
```

Comment:
Created this rollup plugin to preprocess our CSS. 
This is the equivalent for [cssTransformLoader](https://github.com/schrodinger/fixed-data-table-2/blob/9aa115c5797dc314e76963525dd4d383b0fe7696/build_helpers/cssTransformLoader.js) used in our webpack config.